packages/rich-text/src/plugins/Table/index.tsx

```diff
@@ -52,15 +58,35 @@ export const TR = (props: Slate.RenderElementProps) => (
   </tr>
 );
 
-export const TD = (props: Slate.RenderElementProps) => (
-  <td
-    {...props.attributes}
-    // may include `colspan` and/or `rowspan`
-    {...(props.element.data as TableCell['data'])}
-    className={styles[BLOCKS.TABLE_CELL]}>
-    {props.children}
-  </td>
-);
+export const TD = (props: Slate.RenderElementProps) => {
+  const editor = useStoreEditor();
+  const selectedNode = useAnchorNode();
```

Comment:
Why do we need this for? Can't we use some utils from SlateJS?

Reply:
Are you referring to `useAnchorNode` hook? that's just a helper to get the current _HTML_ Node under the cursor. If you have a better way to detect the focused cell let me know. Added a little comment on how it works.

> Can't we use some utils from SlateJS?

I'm not aware of any but just be clear. the Node here is an HTML node, not Slate's :)

Reply:
Oh, now I know what you mean 😄 . There is `Slate.useSelect()` 🤦🏻‍♂️